contracts/Gateway.sol

```diff
@@ -101,14 +107,16 @@ contract Gateway is ProtocolVersioned, Owned {
         workers = _workers;
         bounty = _bounty;
         uuid = _uuid;
-
+        conversionRate = _conversionRate;
+        conversionRateDecimals = _conversionRateDecimals;
     }
 
     /**
      *  @notice External function requestStake.
      *
      *  @dev In order to request stake the staker needs to approve Gateway contract for stake amount.
      *       Staked amount is transferred from staker address to Gateway contract.
+     *   Only part of stake amount is consumed which can minted to BT without any loss
```

Comment:
I think this statement is misleading, because (a) it's possible that all of the amount will be consumed—a clunkier, but accurate, description would be the _maximum stakeable amount given the conversion rate_ (and specifically not `conversionRate`) is consumed; and (b) this is not loss-less, but the amount of loss is at most 1 OST Wei, which we think is acceptable. Additionally, I think it might be confusing that the documentation refers to "stake[d] amount" now that the event includes an element, `_stakedAmount`. Suggest referring to "requested stake amount" or "requested amount" instead; please update the entirety of the `@dev` for this function accordingly.

Please also align this last sentence with the rest of the `@dev` comment.

Reply:
Done 👍  
However, I wanted to discuss the point related to 'loss-less'.